.rubocop.yml

```diff
@@ -47,7 +47,7 @@ Metrics/ClassLength:
   Max: 200
 
 Metrics/ParameterLists:
-  Max: 10
+  Max: 15
```

Comment:
@zverok - Acknowledged. Combined CSV stdlib options like `:col_sep`, `:converters`, etc. to `**options` with default values. It's now back to < 10.